lib/pubsub/index.js

```diff
@@ -483,11 +483,16 @@ PubSub.prototype.subscribe = function(topic, subName, options, callback) {
  * @param {string} name - The name of the subscription.
  * @param {object=} options - Configuration object.
  * @param {boolean} options.autoAck - Automatically acknowledge the message once
- *     it's pulled.
+ *     it's pulled. (default: false)
  * @param {string} options.encoding - When pulling for messages, this type is
  *     used when converting a message's data to a string. (default: 'utf-8')
  * @param {number} options.interval - Interval in milliseconds to check for new
- *     messages.
+ *     messages. (default: 10)
+ * @param {number} options.maxInProgress - Maximum messages to consume
```

Comment:
In case you missed it, the last word might be the most important, which is on the next line. The full sentence is "Maximum messages to consume simultaneously."